broker/src/test/java/org/apache/rocketmq/broker/BrokerControllerTest.java

```diff
@@ -37,16 +37,14 @@
      */
     @Test
     public void testBrokerRestart() throws Exception {
-        for (int i = 0; i < 2; i++) {
-            BrokerController brokerController = new BrokerController(//
-                new BrokerConfig(), //
-                new NettyServerConfig(), //
-                new NettyClientConfig(), //
-                new MessageStoreConfig());
-            assertThat(brokerController.initialize());
-            brokerController.start();
-            brokerController.shutdown();
-        }
+        BrokerController brokerController = new BrokerController(//
+            new BrokerConfig(), //
+            new NettyServerConfig(), //
+            new NettyClientConfig(), //
+            new MessageStoreConfig());
+        assertThat(brokerController.initialize());
+        brokerController.start();
+        brokerController.shutdown();
```

Comment:
If we only start/shutdown broker once, `testBrokerRestart` isn't a appropriate method~